pkg/servicequota/servicequota.go

```diff
@@ -0,0 +1,89 @@
+// Package servicequota check if you have enough resources in aws before cluster creation starts
+package servicequota
+
+import (
+	"encoding/json"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/service/servicequotas"
+	"github.com/hjson/hjson-go"
+	"github.com/oslokommune/okctl/pkg/api/okctl.io/v1alpha1"
+	"github.com/tidwall/gjson"
+)
+
+// Usage defines what we need to know about a service quota
+type Usage interface {
```

Comment:
Maybe we can simplify this interface, seems we only need `CheckAvailability`? Also, it is idiomatic to name the interface with an `er` extension (https://golang.org/doc/effective_go.html#interface-names), so maybe `Checker` is a better name than `Usage`?

Reply:
Agree, done :+1: